### PR TITLE
Add access to methods

### PR DESCRIPTION
**Pull request by yotamdaniel**

### CODE REVIEW

File: test/ArchToken.ts [hunk 1]
```diff
@@ -132,10 +134,10 @@ describe("Arch Token test suit", function () {
 
             it("Should NOT decrease the spender allowance", async function () {
                 // transfer 1 eth
-                await r.archToken.transferFrom(r.addr1.address, r.addr2.address, amount1);
+                await r.archToken.transferFrom(r.addr1.address, r.addr2.address, ethers.utils.parseUnits("1"));
```
Comment (yotamdaniel) [inline test/ArchToken.ts:137]: change back to amount1

File: test/ContractTestContext.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Contract } from "ethers";
+import { BigNumber, Contract } from "ethers";
```
Comment (yotamdaniel) [inline test/ContractTestContext.ts:1]: Remove

File: test/Coordinator.ts [hunk 3]
```diff
@@ -22,7 +27,7 @@ describe("Coordinator Test suit", function () {
 
     before(async function () {
         r = await buildContractTestContext();
-
+        setCoordinatorAsExcecutive(r);
```
Comment (yotamdaniel) [inline test/Coordinator.ts:30]: Add await

File: test/Exchanger.ts [hunk 4]
```diff
@@ -37,6 +37,9 @@ describe("Exchanger Test suit", function () {
         await lvUSD.mint(amountStarting);
         await helperSwapETHWithOUSD(owner, amountStarting);
         await ousd.transfer(exchanger.address, amountStarting);
+
+        // Note that owner is set as executive of exchanger as deafult.
```
Comment (yotamdaniel) [inline test/Exchanger.ts:41]: Remove

File: test/LeverageEngine.ts [hunk 5]
```diff
@@ -72,6 +74,7 @@ describe("LeverageEngine test suit", async function () {
             await prepForPositionCreation();
         });
         it("Should emit position creation event", async function () {
+            // console.log("actual treasury: await r.parameterStore.getTreasuryAddress()");
```
Comment (yotamdaniel) [inline test/LeverageEngine.ts:77]: Remove

**Review by jasonstockman — APPROVED** [review]

**Pull request merged**